reporter/src/test/kotlin/utils/FreeMarkerTemplateProcessorTest.kt

```diff
@@ -183,35 +215,76 @@ class FreeMarkerTemplateProcessorTest : WordSpec({
         }
     }
 
-    "mergeResolvedLicenses" should {
-        "merge the original expressions by license sources" {
-            val resolvedLicenses = listOf(
-                ResolvedLicense(
-                    license = "MIT".toSpdx() as SpdxSingleLicenseExpression,
-                    originalDeclaredLicenses = emptySet(),
-                    originalExpressions = mapOf(LicenseSource.DECLARED to setOf("MIT".toSpdx())),
-                    locations = emptySet()
-                ),
-                ResolvedLicense(
-                    license = "MIT".toSpdx() as SpdxSingleLicenseExpression,
-                    originalDeclaredLicenses = emptySet(),
-                    originalExpressions = mapOf(LicenseSource.DECLARED to setOf("GPL-2.0-only OR MIT".toSpdx())),
-                    locations = emptySet()
-                )
+    "mergeLicenses" should {
+        "merge the expressions from packages by license sources" {
+            val projects = ORT_RESULT.getProjects().toList()
```

Comment:
Maybe add a code comment here that you only convert the set to a list for random access?

Reply:
Similar below.

Reply:
Rather than adding a comment and repeating it multiple times, I extracted a function to return the test projects as a list. This function has a corresponding comment.